js/dialogcards.js

```diff
@@ -70,26 +92,138 @@ H5P.Dialogcards = (function ($, Audio, JoubelUI) {
       '<div class="h5p-dialogcards-description">' + self.params.description + '</div>'
       ));
 
+      // If we are resuming task from a previously finished task, ask user it they want to Retry.
+      if (this.taskFinished) {
+        self.finishedScreen();
+        return;
+      }
+      var existsCardOrder = true;
+      if ($.isEmptyObject(this.cardOrder)) {
+        existsCardOrder = false;
+      }
+      // Create cardOrder and cardNumber buttons only on first instanciation for logged in user.
+      if (self.params.behaviour.randomCards == 'user' && !existsCardOrder) {
+        self.createOrder().appendTo(self.$inner);
+      } else {
+        self.attachContinue();
+      }
+  };
+
+    /**
+   * Attach the rest of the h5p inside the given container.
+   *
+   * @param {jQuery} $container
+   */
+   C.prototype.attachContinue = function ($container) {
+    var self = this;
+
+    // Remove potential user interaction elements from DOM.
+    var $el = '.h5p-dialogcards-number';
+    $( $el ).remove();
```

Comment:
Hi, I created an issue regarding jQuery and H5P https://h5ptechnology.atlassian.net/browse/HFP-874

If we stop having jQuery in H5P core the implication for devs who want to use jQuery will be that they get a dependency to an H5P jQuery library and eventually the attach function will probably be removed in favour of one that returns a DOM object.